app/assets/stylesheets/widgets/home_page/hero_panel.scss

```diff
@@ -21,7 +21,7 @@
   clear: both;
   position: relative;
   overflow: hidden;
-  min-height: 300px;
+  height: 300px;
```

Comment:
I mean I guess they explain it in a sense. Those min-height settings were certainly the root of the problem there, but I'm still a little miffed as to why they weren't causing issues before.